.ci/test_r_package.sh

```diff
@@ -101,9 +101,9 @@ fi
 # to avoid a CI-time dependency on devtools (for devtools::install_deps())
 packages="c('data.table', 'jsonlite', 'Matrix', 'R6', 'testthat')"
 if [[ $OS_NAME == "macos" ]]; then
-    packages+=", type = 'binary'"
+    packages+=", type = 'both'"
```

Comment:
as a reminder, this flag cannot be used on Linux, and that's why it's in this `if`.

From inside a `rocker/verse` container:

```text
> install.packages("jsonlite", type = "both")
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Error in install.packages("jsonlite", type = "both") : 
  type == "both" can only be used on Windows or a CRAN build for macOS
```